workshops/forms.py

```diff
@@ -1080,3 +1107,244 @@ class Meta:
             'domains': forms.CheckboxSelectMultiple(),
             'lessons': forms.CheckboxSelectMultiple(),
         }
+
+
+class TrainingProgressForm(forms.ModelForm):
+    trainee = selectable.AutoCompleteSelectField(
+        lookup_class=lookups.PersonLookup,
+        label='Trainee',
+        required=True,
+        help_text=AUTOCOMPLETE_HELP_TEXT,
+        widget=selectable.AutoComboboxSelectWidget,
+    )
+    awarded_by = selectable.AutoCompleteSelectField(
+        lookup_class=lookups.AdminLookup,
+        label='Awarded by',
+        required=True,
+        help_text=AUTOCOMPLETE_HELP_TEXT,
+        widget=selectable.AutoComboboxSelectWidget,
+    )
+    event = selectable.AutoCompleteSelectField(
+        lookup_class=lookups.TTTEventLookup,
+        label='Event',
+        required=False,
+        help_text=AUTOCOMPLETE_HELP_TEXT,
+        widget=selectable.AutoComboboxSelectWidget,
+    )
+
+    # In the form, "----" option is displayed for requirement even though
+    # it's required non-blank field. This is a new bug in Django. See
+    # https://code.djangoproject.com/ticket/26813.
```

Comment:
I've just noticed that this Django bug is actually about RadioSelect, not select widget:

> Unlike the select widget, where a blank option is idiomatic even for required fields, ...

So we can actually rely on this behaviour.